### PR TITLE
docs: clarify and add Interpolate src/kw examples

### PR DESCRIPTION
**Pull request by Frodox**
## Contributor Checklist:

* [ ] I have updated the unit tests
* [ ] I have created a file in the master/buildbot/newsfragment directory (and read the README.txt in that directory)
* [ ] I have updated the appropriate documentation

### CODE REVIEW

File: master/docs/manual/cfg-properties.rst [hunk 1]
```diff
@@ -234,9 +235,28 @@ The following ways of interpreting the value are available.
...
+            util.Interpolate('-b %(src::branch)s'),
+            util.Interpolate('-d %(kw:data)s', data="some extra needed data")
+        ]))
+
+.. note::
+
+   We use ``%(src::branch)s`` in most of examples, because ``codebase`` is empty by default.
+
+
+Example::
```
Comment (sa2ajj) [inline master/docs/manual/cfg-properties.rst]: please use explicit `.. code-block: python`, we had issues when automatic detection of the language did not work (while you are at it, please apply the same change to `Example::` above).

File: master/docs/manual/cfg-properties.rst [hunk 2]
```diff
@@ -234,9 +235,28 @@ The following ways of interpreting the value are available.
...
+            'save-build-artifacts-script.sh',
+            util.Interpolate('-r %(prop:repository)s'),
+            util.Interpolate('-b %(src::branch)s'),
+            util.Interpolate('-d %(kw:data)s', data="some extra needed data")
+        ]))
+
+.. note::
+
+   We use ``%(src::branch)s`` in most of examples, because ``codebase`` is empty by default.
+
```
Comment (sa2ajj) [inline master/docs/manual/cfg-properties.rst]: we keep *one* empty line between "blocks" in the docs.

Comment (Frodox) [inline master/docs/manual/cfg-properties.rst]: Oh, sorry.. fixed!
It's time to use some docs-guildeline-syntax-checker :)

File: master/docs/manual/cfg-properties.rst (hunk 1 above)
Comment (Frodox) [inline master/docs/manual/cfg-properties.rst]: Yup, fixed

**Comment by tardyp** [comment]
looks good. Please have a look at the CI jobs. notabily, the docs actually does not build, because of indentation

**Comment by Frodox** [comment]
> Please have a look at the CI jobs. notabily, the docs actually does not build, because of indentation

oh.. rst has a bit strange syntax rules for code-block =\ Ok, I'll take a look!

File: master/docs/manual/cfg-properties.rst [hunk 3]
```diff
@@ -233,10 +234,32 @@ The following ways of interpreting the value are available.
 
 Example::
```
Comment (tardyp) [inline master/docs/manual/cfg-properties.rst]: remove the ``::``

**Comment by tardyp** [comment]
thanks!

**Pull request merged**